containers/nestjs/src/app.gateway.ts

```diff
@@ -114,4 +120,11 @@ export class AppGateway {
       game.movePaddle(game._rightPlayer.paddle, keydown, north)
     }
   }
+
+  @SubscribeMessage('code')
```

Comment:
I see now sort of what you guys were trying to achieve, the problem here is that I also need a response, to see whether the authentication was successful. So my suggestion is to instead of making a new event, creating a endpoint, like '/api/authenitcate'

Reply:
another approuch would be to create routes with middleware that protects the routes. I'm more in favor of my first suggestion tough, as that would demand less of a big rewrite most probable. But I haven't looked into it that deep.